libs/sdk-core/src/breez_services.rs

```diff
@@ -1064,20 +1064,26 @@ impl BreezServices {
     /// An error is thrown if the log file cannot be created in the working directory.
     ///
     /// An error is thrown if a global logger is already configured.
-    pub fn init_logging(log_dir: &str, app_logger: Option<Box<dyn log::Log>>) -> SdkResult<()> {
-        let target_log_file = Box::new(
-            OpenOptions::new()
-                .create(true)
-                .append(true)
-                .open(format!("{log_dir}/sdk.log"))
-                .map_err(|_| SdkError::InitFailed {
-                    err: "Can't create log file".into(),
-                })?,
-        );
-        let logger = env_logger::Builder::new()
-            .target(env_logger::Target::Pipe(target_log_file))
-            .parse_filters(
-                r#"
+    pub fn init_logging(
```

Comment:
NIT: The description for this has to be updated, but otherwise looks good.

Reply:
My mistake, above comment is fine.

I meant the one in uniffi has to be updated: https://github.com/breez/breez-sdk/blob/c875ce74a123da5caf4c58a5383978ac919f30a5/libs/sdk-bindings/src/uniffi_binding.rs#L61-L62